.github/workflows/test.yml

```diff
@@ -26,6 +26,34 @@ jobs:
             fi
           done
           exit $errors
+  runSonarCloud:
+    name: SonarCloud for ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: "Project Root"
+            dir: "."
+          - name: "API Package"
+            dir: "packages/api"
+          - name: "Cognito Package"
+            dir: "packages/cognito"
+          - name: "Infrastructure"
+            dir: "infrastructure"
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Trigger SonarCloud scan
+        uses: SonarSource/sonarcloud-github-action@master
+        with:
+          projectBaseDir: ${{ matrix.dir }}
+          args: |
+            -Dsonar.typescript.tsconfigPath=/github/workspace/tsconfig.json
```

Comment:
This will override the value(s) you're specifying directly in the files

Reply:
I did this to try to let us use a shared tsconfig (which seems to no longer be feasible)